README.md

```diff
@@ -54,7 +62,11 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+def Belle
+  return "#{residents,1}"
+end
+def residents<< 'Belle'
+
```

Comment:
not quite, we just want to use `def` for method definitions. Here we are just going to call methods on existing variables.

```ruby
town[:residents].delete("Belle")
town[:castle][:guests] << "Belle"